Multisig/Logic/Models/KeyInfo.swift

```diff
@@ -393,6 +393,34 @@ extension KeyInfo {
         if let key = (try? privateKey()) { return key }
         return try? delegatePrivateKey()
     }
+
+    func pushNotificationSigningKey(completion: @escaping (Result<PrivateKey?, Error>) -> ()) {
+        privateKey { [unowned self] result in
+            switch result {
+            case .success(let privateKey):
```

Comment:
I think this is wrong, 

pushNotificationSigningKey should return the delegate key for all key types now right? not the private key